src/schema/draft/draft_utils_manifestations.js

```diff
@@ -479,13 +508,26 @@ export async function resolveOnlineAccess(pid, context) {
   const data = getArray(manifestation, "details.onlineAccess");
   data.forEach((entry) => {
     if (entry.value) {
+      // hold origin
+      const tmpOrigin = parseOnlineUrlToOrigin(
+        (entry.value.link && entry.value.link.$) || ""
+      );
+
```

Comment:
Bare et forslag, som muligvis kan forbedre læsbarhed. Ignorer, hvis du ikke er enig.
omdøb getRealUrl til getProxyUrl. Ryk tjek for ebookCentral ud fra getRealUrl funktion. Lav en ny const der hedder proxyRequired. Så kan man også slippe for at tjekke for ebookcentral flere gange.